src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminOrganizationResource.java

```diff
@@ -0,0 +1,289 @@
+package de.tum.in.www1.artemis.web.rest.admin;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import de.tum.in.www1.artemis.domain.Organization;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.repository.CourseRepository;
+import de.tum.in.www1.artemis.repository.OrganizationRepository;
+import de.tum.in.www1.artemis.repository.UserRepository;
+import de.tum.in.www1.artemis.security.annotations.EnforceAdmin;
+import de.tum.in.www1.artemis.service.OrganizationService;
+import de.tum.in.www1.artemis.web.rest.OrganizationResource;
+import de.tum.in.www1.artemis.web.rest.dto.OrganizationCountDTO;
+import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
+import de.tum.in.www1.artemis.web.rest.util.HeaderUtil;
+
+/**
+ * REST controller for administrating the Organization entities
+ */
+@RestController
+@RequestMapping("api/admin/")
+public class AdminOrganizationResource {
+
+    private final Logger log = LoggerFactory.getLogger(OrganizationResource.class);
```

Comment:
❌ Required Change: The logger seems to refer to the wrong class. Please check the other occurrences also